gateway-service-definitions/src/main/resources/services/hdfsui/3.0.0/service.xml

```diff
@@ -16,6 +16,12 @@
    limitations under the License.
 -->
 <service role="HDFSUI" name="hdfs" version="3.0.0">
+    <metadata>
+        <type>UI</type>
+        <context>/hdfs?host={{BACKEND_HOST}}</context>
```

Comment:
should be `/hdfs/?host=` notice the trailing slash after `/hdfs`

Reply:
Fixed.